install.py

```diff
@@ -204,13 +204,13 @@ def ensure(pip=None, pyvenv=None):
                    % sys.executable))
 
 
-def get_requirements_branch(version, inst, repository):
+def get_requirements_branch(version, inst, from_fork):
     """
     Convert "version" into a branch name
     """
-    repository_owner, repository_name = repository.split('/')
-    # in forks of oq-engine, always read requirements from master
-    if repository_owner != 'gem' and repository_name == 'oq-engine':
+    # in actions triggered by forks we want requirements to be taken from
+    # master
+    if from_fork:
```

Comment:
Perhaps it is possible to do a less invasive change and look at the environment variable set by the actions, something like `if os.environ.get('FROM_FORK')`, however I am fine also with the current approach.

Reply:
As agreed, I suppressed the help for the --from_fork flag, so it is less invasive